spec/adapters/mysql_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Polo::Adapters::MySQL do
+  include Polo::Adapters::MySQL
+
+  let(:netto) do
+    AR::Chef.where(name: 'Netto').first
+  end
+
+  before(:all) do
+    TestData.create_netto
+  end
+
+  let(:translator) { Polo::SqlTranslator.new(netto, Polo::Configuration.new(adapter: :mysql)) }
+
+  describe '#ignore_transform' do
+    it do
```

Comment:
let's make sure we add a title for this test.
`it 'appends the IGNORE command after INSERTs'`